client/src/templates/Challenges/utils/index.js

```diff
@@ -6,5 +6,5 @@ export function createGuideUrl(slug = '') {
 
 export function isGoodXHRStatus(status) {
   const statusInt = parseInt(status, 10);
-  return statusInt >= 200 && statusInt < 400;
+  return (statusInt >= 200 && statusInt < 400) || statusInt === 402;
```

Comment:
Is 402 a good status? When are we getting that back? I assume it is from paypal.

Reply:
We ourselves are returning this from the api for valid stripe errors